web-ui/src/main/resources/catalog/views/default/templates/advancedSearchForm/defaultAdvancedSearchForm.html

```diff
@@ -1,4 +1,9 @@
-<div class="gn-search-filter col-md-offset-3 col-md-9 advancedSearchForm">
+<div class="gn-search-filter advancedSearchForm">
```

Comment:
Not sure if we can better align options & autocomplete using sometime square or round corners ?

![image](https://user-images.githubusercontent.com/1701393/127276815-51c9da2d-c94a-4916-9f67-2d9226451158.png)

Reply:
Yes, we can definitely align it better. Going to improve it